### DISCUSSION TITLE
content/how-to-use-public-domain-comics-for-cherokee-language-teaching/

### DISCUSSION THREAD

**Discussion by giscus** [CherokeeLanguage/CherokeeLessons.com#19] [General]
# content/how-to-use-public-domain-comics-for-cherokee-language-teaching/

https://www.cherokeelessons.com/content/how-to-use-public-domain-comics-for-cherokee-language-teaching/

**Comment by woodygeo** [comment]
I lead a local study group for Cherokee language beginners, and the whole group really appreciates the comics. It helps add good context, words, pictures, action, humor. A really effective teaching tool for all ages. I look forward to using Migali's techniques to do some comics like this.
If it works for Tsalagi, it can work for your language too.
